server/chat-commands/admin.ts

```diff
@@ -415,6 +415,26 @@ export const commands: Chat.ChatCommands = {
 		`If a [highlight] is specified, only highlights them if they have that term on their highlight list.`,
 	],
 
+	botmsg(target, room, user, connection) {
+		if (!target || !target.includes(',')) return this.parse('/help botmsg');
+		target = this.splitTarget(target);
+		const targetUser = this.targetUser;
+		const targetUsername = this.targetUsername;
```

Comment:
Would it make more sense to make `splitTarget` return these two variables (or something like `{newTarget, targetUser, targetUsername}`), similar to how datsearch's `getMod` function returns the mod and adjusted target, or similar to how `extractFormat()` is used? Would probably be for another PR, but it's used this way in a lot of other commands.

Reply:
You mean `splitFormat`, and yes, I think `splitTarget` sucks and should be refactored.

Reply:
We can call it `splitUser`.